README.md

```diff
@@ -1 +1,966 @@
+
+```bash
+   $ kubectl create -f ./kubernetes/
+```
+
+- Now let's see how we can deploy the `phone_store_service` on Kubernetes. We need to import `` ballerinax/kubernetes; `` and use ``@kubernetes``annotations as shown below to enable kubernetes deployment.
+
+#####  phone_store_service.bal
+
+```ballerina
+import ballerinax/kubernetes;
+// Other imports
+
+// Type definition for a phone order
+
+json[] phoneInventory = ["Apple:190000", "Samsung:150000", "Nokia:80000", "HTC:40000", "Huawei:100000"];
+
+// 'jms:Connection' definition
+
+// 'jms:Session' definition
+
+// 'jms:QueueSender' endpoint definition
+
+@kubernetes:Ingress {
+hostname:"ballerina.guides.io",
+name:"ballerina-guides-phone_store_service",
+path:"/"
+}
+
+@kubernetes:Service {
+serviceType:"NodePort",
+name:"ballerina-guides-phone_store_service"
+}
+
+@kubernetes:Deployment {
+image:"ballerina.guides.io/phone_store_service:v1.0",
+name:"ballerina-guides-phone_store_service"
+}
+
+endpoint http:Listener listener {
+port:9090
+};
+
+@http:ServiceConfig {basePath:"/phonestore"}
+service<http:Service> phone_store_service bind listener {
+``` 
+
+- Here we have used ``  @kubernetes:Deployment `` to specify the Docker image name which will be created as part of building this service. 
+- We have also specified `` @kubernetes:Service `` so that it will create a Kubernetes service, which will expose the Ballerina service that is running on a Pod.  
+- In addition we have used `` @kubernetes:Ingress ``, which is the external interface to access your service (with path `` /`` and host name ``ballerina.guides.io``)
+
+- Now you can build a Ballerina executable archive (.balx) of the service that we developed above, using the following command. This will also create the corresponding Docker image and the Kubernetes artifacts using the Kubernetes annotations that you have configured above.
+  
+```
+   $ ballerina build 
+   
+   @kubernetes:Service                      - complete 1/1
+   @kubernetes:Ingress                      - complete 1/1
+   @kubernetes:Docker                       - complete 3/3 
+   @kubernetes:Deployment                   - complete 1/1
+  
+   Run following command to deploy kubernetes artifacts:  
+   kubectl apply -f ./target/phone_store_service/kubernetes
+   
+   @kubernetes:Service                      - complete 1/1
+   @kubernetes:Ingress                      - complete 1/1
+   @kubernetes:Docker                       - complete 3/3 
+   @kubernetes:Deployment                   - complete 1/1
+  
+   Run following command to deploy kubernetes artifacts:  
+   kubectl apply -f ./target/phone_order_delivery_service/kubernetes
+  
+   
+```
+
+- You can verify that the Docker image that we specified in `` @kubernetes:Deployment `` is created, by using `` docker images ``. 
```

Comment:
rephrase to "Use the `docker images` command to verify whether the Docker image that you specified in `@kubernetes:Deployment` was created."